source/fundamentals/crud/write-operations/embedded-arrays.txt

```diff
@@ -50,7 +50,7 @@ arrays, see our :ref:`guide on the Updates builder <array_updates>`.
 The following example performs these actions: 
 
 - Query for the sample document
-- Add "17" to the ``qty`` array using the ``push()`` method of the ``Updates`` builder
+- Append "17" to the ``qty`` array for document that matches the query filter
```

Comment:
Nitpick: I think "in" might be a better preposition than "for" and it's missing an article ("the").